pkgs/applications/editors/jetbrains/plugins/specialPlugins.nix

```
@@ -4,17 +4,17 @@
 {
   "631" = {
     # Python
-    nativeBuildInputs = [ autoPatchelfHook ];
+    nativeBuildInputs = lib.optionals stdenv.isLinux [ autoPatchelfHook ];
```

This and the other spots where this occurs in this file can be simplified with

```suggestion
    nativeBuildInputs = lib.optional stdenv.isLinux autoPatchelfHook;
```